aerogear-android-push-test/src/main/AndroidManifest.xml

```diff
@@ -28,11 +28,6 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />    
     <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
```

Comment:
let's remove this too ? (see https://developers.google.com/cloud-messaging/android/android-migrate-fcm#remove_the_permissions_required_by_gcm )